### PR TITLE
Add request to form initialization and update related tests

### PR DESCRIPTION
**Pull request by dwasyl**
Adds `request` to the `ElevateForm` initialization, and associated references in `ElevateView`. Also updated all references in existing tests.

### CODE REVIEW

File: elevate/forms.py [hunk 1]
```diff
@@ -18,12 +18,13 @@ class ElevateForm(forms.Form):
...
-    def __init__(self, user, *args, **kwargs):
+    def __init__(self, request, user, *args, **kwargs):
+        self.request = request
         self.user = user
         super().__init__(*args, **kwargs)
 
     def clean_password(self):
         username = self.user.get_username()
-        if auth.authenticate(username=username, password=self.data['password']):
+        if auth.authenticate(request=self.request, username=username, password=self.data['password']):
```
Comment (moggers87) [inline elevate/forms.py]: This line is too long and that's what's causing flake8 to fail.

**Review by moggers87 — CHANGES_REQUESTED** [review]
Just needs one change to make CI pass and some patch notes so we can do a release (see https://django-elevate.readthedocs.io/en/latest/contributing/index.html#submitting-patches on how that should be done)

File: elevate/forms.py (hunk 1 above)
Comment (dwasyl) [reply elevate/forms.py]: Sounds good, formatted the line shorter and added the RELEASE doc. Let me know if there's anything else!

**Review by moggers87 — APPROVED** [review]
:+1:

**Pull request merged**